values/demo.yaml

```diff
@@ -157,3 +157,16 @@ abha-verification:
   metadata:
     labels:
       environment: demo
+
+bahmni-metabase:
+  enabled: false
+  config:
+    MB_DB_TYPE: postgres
+    MB_DB_DBNAME: 'metabase'
+    MB_DB_PORT:  5432
+    OPENMRS_DB_TYPE: 'mysql'
+  secrets:
+    MB_ADMIN_FIRST_NAME: 'Admin'
+    MB_DB_USER: 'metabase-user'
+    MB_DB_PASS: 'password'
+    MB_DB_HOST: 'bahmni-dev-postgresql'
```

Comment:
These values are for demo namespace and `bahmni-dev-postgres` exists in dev namespace